opentibiabr/canary
# [Enhancement] - Automatically unzip world.zip in world folder if otbm file doesn't exist on server start

# Description

that's it, simple
you need this library to your vcpkg
./vcpkg install libzippp

need tests and others OS tests (linux and mac)

to test:
remove your .otbm from world folder
remove world.zip from your world folder and check the error in console
## Type of change
